objectherkenning_openbare_ruimte/settings/databricks_jobs_settings.py

```diff
@@ -0,0 +1,26 @@
+import yaml
+
+def load_settings(file_path):
```

Comment:
How come we don't use the settings class?

Reply:
because it uses the CVToolkit repo which I did not manage to install on the cluster in databricks